src/v2/guide/forms.md

```diff
@@ -1,28 +1,28 @@
 ---
-title: Form Input Bindings
+title: Liaisons des champs de formulaire
```

Comment:
@forresst en fait pour être au plus proche de l'original j'aurais du écrire "liaison de champ de formulaire" (singulier). Ca éviterait la confusion dont tu parles et c'est aussi la traduction qui existe aujourd'hui pour les classes et styles : https://fr.vuejs.org/v2/guide/class-and-style.html

Qu'en penses tu ?